compare_test.go

```diff
@@ -97,45 +97,18 @@ type TestStruct struct {
 }
 
 func TestJsonSerialize(t *testing.T) {
-	// a := []TestStruct{TestStruct{left:}}
-	a := []TestStruct{TestStruct{}}
+	s := TestStruct{
+		left:  &TestStructMember{a: "left"},
+		right: &TestStructMember{a: "right"},
+	}
+	a := []TestStruct{s}
 	bytes, err := json.Marshal(a)
 	if err != nil {
 		fmt.Println("e", err)
 	}
 	fmt.Println("s", string(bytes))
 }
 
-func aTestJSONDeserialize(t *testing.T) {
```

Comment:
Looks like Jon was testing this from his Makefile at one point.

Reply:
Do you want to keep it, then?  We could move it into cli_test.go, where the integration tests are.

Reply:
after looking/thinking a little more, let's delete it. 👍